plugins/org.jkiss.dbeaver.ext.oracle/src/org/jkiss/dbeaver/ext/oracle/model/OracleConstants.java

```diff
@@ -86,7 +86,7 @@
     public static final String PROP_METADATA_USE_SYS_SCHEMA = DBConstants.INTERNAL_PROP_PREFIX + "meta-use-sys-schema@";
     public static final String PROP_METADATA_USE_SIMPLE_CONSTRAINTS = DBConstants.INTERNAL_PROP_PREFIX + "meta-use-simple-constraints@";
     public static final String PROP_METADATA_USE_ALTERNATIVE_TABLE_QUERY = DBConstants.INTERNAL_PROP_PREFIX + "meta-use-alternative-table-query@";
-
+    public static final String PROP_SEARCH_METADATA_IN_SYNONYMS = DBConstants.INTERNAL_PROP_PREFIX + "meta-search-in-synonyms@"; //$NON-NLS-1$
```

Comment:
Don't use `DBConstants.INTERNAL_PROP_PREFIX` since it is deprecated. Use plain names for properties without `@` at the end.